pkg/rpcclient/client.go

```diff
@@ -125,7 +128,9 @@ func initClient(ctx context.Context, cl *Client, endpoint string, opts Options)
 	//	if opts.Cert != "" && opts.Key != "" {
 	//	}
 
-	cl.ctx = ctx
+	cancelCtx, cancel := context.WithCancel(ctx)
+	cl.ctx = cancelCtx
```

Comment:
BTW, it's nice fix wrt `Close()` handling.

Reply:
Should we create a separate commit for it?

Reply:
Not worth the trouble.